docs/content/Coding Milestones/PoC2023q1/kubestellar-syncer-subs/kubestellar-syncer-0-deploy-guilder.md

```diff
@@ -2,7 +2,11 @@
 Go to inventory management workspace and find the mailbox workspace name.
 ```shell
 kubectl ws root:imw-1
-mbws=`kubectl get SyncTarget guilder -o jsonpath="{.metadata.annotations['kcp\.io/cluster']}-mb-{.metadata.uid}"`
+kubectl get SyncTargets
+kubectl get synctargets.edge.kcp.io
+kubectl describe synctargets.edge.kcp.io guilder
+#kubectl describe Synctarget guilder
```

Comment:
This is script debugging that should eventually be removed.  Not necessarily in this PR.

Reply:
ok